src/tools/ScenarioMeasurement/Util/Util.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace ScenarioMeasurement
+{
+
+    public class Util
+    {
+        [DllImport("User32.Dll")]
+        private static extern long SetCursorPos(int x, int y);
```

Comment:
I don't think you should block on this, but one thing we should consider is how we're going to handle platform specific stuff.  Right now, the system depends on an ETL, but you could imagine it working with an eventpipe file or LTTng file.  There will always be some stuff like this pinvoke that are platform specific and so we should think about how we want to handle these things (statically at compile time vs. dynamically at runtime).

Reply:
Yup.